components/notebook-controller/config/crds/notebook_v1alpha1_notebook.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: notebooks.notebook.kubeflow.org
+spec:
+  group: notebook.kubeflow.org
+  names:
+    kind: Notebook
+    plural: notebooks
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
```

Comment:
for the ksonnet package I had provided a notebook schema which is basically the PodTemplateSpec schema, might be nice to have something which can be validated